pyproject.toml

```diff
@@ -19,7 +19,7 @@ classifiers = [
     "Topic :: Multimedia :: Sound/Audio",
 ]
 dependencies = [
-    "pipecat-ai>=0.0.67",
+    "pipecat-ai>=0.0.73",
```

Comment:
Note: this is just a guess as to the version of pipecat-ai that will contain `BaseDirectFunctionWrapper`. I'll make sure to update this to the appropriate version if it seems like it'll differ from the one named here.

Reply:
Bumped the version to 0.0.74 now